### PR TITLE
feat: re-enable TrustyAI in downstream but set to removed (T.P)

### PR DESCRIPTION
**Pull request by zdtsw**
Revert "Remove Trustyai installation in downstream" 
set version to 2.11.0
set default trustyai to removed

This reverts commit 9392209531ffaa80429f17aa19c3b6e5c5774efc.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -239,11 +233,6 @@ func main() {
 		setupLog.Error(err, "error remove deprecated resources from previous version")
 	}
 
-	// Apply update from legacy operator
-	if err = upgrade.UpdateFromLegacyVersion(setupClient, platform, dscApplicationsNamespace, dscMonitoringNamespace); err != nil {
```
Comment (ykaliuta) [inline main.go:243]: Is it part of this patch?

**Comment by ykaliuta** [comment]
How about https://github.com/red-hat-data-services/rhods-operator/blob/main/controllers/datasciencecluster/datasciencecluster_controller.go#L352 ?

File: main.go (hunk 1 above)
Comment (zdtsw) [reply main.go]: follow up in https://github.com/red-hat-data-services/rhods-operator/pull/284

**Review by ykaliuta — APPROVED** [review]
I would prefer setting version separately, but ok.

**Pull request merged**